### PR TITLE
IOS-4422: Created ExtensionQuote

### PR DESCRIPTION
**Pull request by acmaciek**
**Issue Link**
https://spothero.atlassian.net/browse/IOS-4422

**Description**
Created new **ExtensionQuote** model that will include 

Sample response:
https://documenter.getpostman.com/view/13573064/2s9YJaXPP4

Documentation:
https://spothero.atlassian.net/wiki/spaces/~253418099/pages/3032383788/Proposal+-+Update+searchTransientFacilityParking+API+to+support+the+extension+upselling

### CODE REVIEW

**Comment by mikepoly** [comment]
@acmaciek This pr https://github.com/spothero/SpotHeroSDK-iOS/pull/109 should fix the tests and unblock you. You will need to pull this code from main.

File: Sources/SpotHeroAPI/Search/Transient/Models/TransientRateContainer.swift [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+
     /// A quote for parking provided by SpotHero to a prospective renter.
     /// This quote is generated by SpotHero, includes detailed pricing information,
     /// and is guaranteed to be honored for the period of validity specified within
     /// so long as no part of the originally issued quote has been modified.
     public let quote: Quote
+
+    /// JSON object that will have the extended rate quote
+    /// It will include the extend_hours and the price_difference.
+    public let extensionQuotes: [ExtensionQuote]
```
Comment (delannoyk) [inline Sources/SpotHeroAPI/Search/Transient/Models/TransientRateContainer.swift:22]: This should be optional

Comment (acmaciek) [reply Sources/SpotHeroAPI/Search/Transient/Models/TransientRateContainer.swift:22]: Ahhh facts !

**Review by delannoyk — APPROVED** [review]
Looks good!

**Review by delannoyk — APPROVED** [review]

**Pull request merged**